src/components/socialLink/SocialLinks.component.tsx

```diff
@@ -0,0 +1,19 @@
+import { socialNetworks } from '../../constants/socialNetworks';
+
+import styles from './socialLink.module.css';
+
+function SocialLinks() {
+    return (
+        <ul className={styles.links}>
+            {socialNetworks.map((network) => (
+                <li key={network.name}>
```

Comment:
if you did task 1*. You need to create another component `SocialLink` with own props.

Reply:
Done